psalm.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<psalm
+        errorLevel="2"
```

Comment:
Same here. The level might be too low to start with.

Reply:
I didn't know that 8 is the lowest. I was expecting similar functionality as phpstan. Updated.